fossunited/fossunited/web_template/events_timeline/events_timeline.html

```diff
@@ -79,84 +90,97 @@
 <div v-scope>
   <div class="container events-timeline">
     <!--Search filter panel-->
-    <div class="search-filter-controlpanel">
+    <div class="search-filter-controlpanel" role="search" aria-labelledby="search-panel-label">
+      <h2 id="search-panel-label" class="sr-only">Search and Filter Events</h2>
+
+      <!-- Search Input -->
       <div class="events-search">
+        <label for="events-search" class="sr-only">Search Events</label>
         <input
           v-model="searchText"
           type="text"
           id="events-search"
           placeholder="Search by event name and date"
+          aria-label="Search events by name and date"
         />
       </div>
 
+      <!-- Must Attend Checkbox -->
       <div class="must-attend-checkbox">
-        <input type="checkbox" v-model="mustAttendOnly" />
+        <input 
+          type="checkbox" 
+          v-model="mustAttendOnly" 
```

Comment:
```suggestion
        <input
          type="checkbox"
          v-model="mustAttendOnly"
```

removed trailing whitespace